testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
I guess that's the point of the change in Haxe 3.2, isn't it?  To get rid of getter/setter methods entirely.  (Well, not really, but to get rid of the independent getters and setters and allow the compiler to generate them based upon the declared access values.)